bootstrap/config.json

```diff
@@ -0,0 +1,389 @@
+{
```

Comment:
This file could safely be deleted, but it also doesn't get loaded by the browser so it doesn't hurt to have it.